import-scripts/subset_and_merge_crdb_pdx_studies.py

```diff
@@ -581,6 +581,7 @@ def annotate_maf(destination_to_source_mapping, root_directory, annotator_jar):
         # otherwise remove annotated maf if exists in destination study directory
         if annotator_status == 0:
             shutil.copy(annot_maf, orig_maf)
+            os.remove(annot_maf)
```

Comment:
I think it would be better/faster/simpler to change:
  shutil.copy(anot_maf, orig_maf)
  os.remove(annot_maf)
to:
  os.replace(annot_maf, orig_maf)